### PR TITLE
Experiments working

### PR DESCRIPTION
**Pull request by RomanSraj**

### CODE REVIEW

File: runs/comparisons/suprb_all_tuning.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def run(problem: str, job_id: str):
 
     estimator = SupRB(
         rule_generation=rs.RandomSearch(),
-        solution_composition=ga.GeneticAlgorithm(),
+        solution_composition=abc.ArtificialBeeColonyAlgorithm(),
```
Comment (heidmic) [inline runs/comparisons/suprb_all_tuning.py:50]: why change the default?

File: runs/comparisons/suprb_all_tuning.py [hunk 2]
```diff
@@ -183,15 +175,15 @@ def suprb_space(trial: Trial, params: Bunch):
...
+        if params.solution_composition == 'GeneticAlgorithm':
+            # GA base
             params.solution_composition = getattr(suprb.optimizer.solution.ga, params.solution_composition)()
-        
-            params.solution_composition__n_jobs = trial.suggest_int('solution_composition__n_jobs', 1, 5)
+
             params.solution_composition__n_iter = trial.suggest_int('solution_composition__n_iter', 16, 64)
             params.solution_composition__population_size = trial.suggest_int('solution_composition__population_size', 16, 64)
-            params.solution_composition__elitist_ratio = trial.suggest_float('solution_composition__elitist_ratio', 0.1, 0.25)
+            params.solution_composition__elitist_ratio = trial.suggest_float('solution_composition__elitist_ratio', 0.0, 0.2)
```
Comment (heidmic) [inline runs/comparisons/suprb_all_tuning.py:186]: highest value should be .3

**Review by heidmic — CHANGES_REQUESTED** [review]

File: runs/comparisons/suprb_all_tuning.py (hunk 1 above)
Comment (RomanSraj) [reply runs/comparisons/suprb_all_tuning.py:50]: Irrelevant as it gets overridden by the tuning. Was changed for testing purposes

File: runs/comparisons/suprb_all_tuning.py (hunk 2 above)
Comment (RomanSraj) [reply runs/comparisons/suprb_all_tuning.py]: Done

**Review by heidmic — APPROVED** [review]

**Pull request merged**